rules/ke_rules.yaml

```diff
@@ -13,89 +13,66 @@
 # limitations under the License.
 
 # Rule Keys:
-# name: The unique name describing the rule
+# name: unique name describing the rule
 # resource: A mapping of resources that this rule applies to.
 #  * type: The type of resource, can be organization, folder, or project.
 #  * resource_ids: A list of one or more numeric ids to match, or '*' for all.
-# check_serverconfig_valid_node_versions: If true, will raise a violation for
-#    any node pool running a version that is not listed as supported for the'
-#    zone the cluster is running in.
-# check_serverconfig_valid_master_versions: If true, will raise a violation for
-#    any cluster running an out of date master version. New clusters can only
-#    be created with a supported master version.
-# allowed_nodepool_versions: Optional, if not included all versions are allowed.
-#        The list of rules for what versions are allowed on nodes.
-#  * major: The major version that is allowed.
-#  * minor: Optional, the minor version that is allowed. If not included, all
-#        minor versions are allowed.
-#  * operator: Optional, defaults to =, can be one of (=, >, <, >=, <=). The
-#        operator determines how the current version compares with the allowed
-#        version. If a minor version is not included, the operator applies to
-#        major version. Otherwise it applies to minor versions within a single
-#        major version.
+# mode: blacklist or whitelist, defaults to whitelist
+# key: jmespath to the values you'd like to check
+# values: a list of values to blacklist/whitelist according to mode
 
 rules:
-# The default KE rule alerts if any node pool is running a version of
-# kubernetes that is no longer supported
-  - name: Unsupported node pool version
+  - name: logging should be enabled
     resource:
-      - type: organization
+      - type: project
         resource_ids:
           - '*'
-    check_serverconfig_valid_node_versions: true
+    key: loggingService
+    values:
+      - logging.googleapis.com
 
-    # Set check_serverconfig_valid_master_versions to true to check if the
-    # master version is supported as well. Newly created clusters can only use
-    # one of the supported master versions.
-    # Master versions are always greater or equal to the node pool version.
-    check_serverconfig_valid_master_versions: false
+# # Example rules:
```

Comment:
You have some more complex examples in your tests, it would be helpful to include them here too. Especially rules with list expansion for repeated fields.

Reply:
I'll add a few more examples and a pointer to the test data.  The installer uses `str.format` to fill in the templates, so it breaks if the rules contain `{...}` expressions.